### ISSUE TITLE
Add to the Search, which is called by pressing the Ctrl + Shift + F key, the Replace option.

### ISSUE THREAD

**Issue by youra-h** [microsoft/vscode#138585]
The castrated window on the control panel is extremely inconvenient

**Comment by IllusionMH** [comment]
/question

It was these from the beginning. You just need to click on `>` near to search field or run `> Search: Replace in Files`

**Comment by youra-h** [comment]
> /question
> 
> It was these from the beginning. You just need to click on `>` near to search field or run `> Search: Replace in Files`

Sorry! But I cannot find it. First of all, I poked at all the available buttons, and then I created a task. I will attach screenshots.
There is
![image](https://user-images.githubusercontent.com/5162766/145028401-c5e37542-6f9d-4e35-98e2-6c3a58ba5cc4.png)
If you click on a folder, then a menu will appear in which there is only the item Search in folder
![image](https://user-images.githubusercontent.com/5162766/145028862-4e3e6d40-80c9-495a-b241-4720421151bc.png)
Searching in a large window, I did not find the Replace button
![image](https://user-images.githubusercontent.com/5162766/145029096-d2b50f21-1122-4e56-a671-dd89e9d3f6ff.png)
